lib/mapping.gd

```diff
@@ -695,6 +695,29 @@ DeclareAttribute( "IdentityMapping", IsCollection );
 ##
 DeclareAttribute( "InverseGeneralMapping", IsGeneralMapping );
 
+#############################################################################
+##
+#A  RestrictedInverseGeneralMapping( <map> )
+##
+##  <#GAPDoc Label="RestrictedInverseGeneralMapping">
```

Comment:
Is there a matching `Include` statement somewhere in `doc/ref/*.xml`?

Reply:
added now.